app/uk/gov/hmrc/apisubscriptionfields/config/microserviceWiring.scala

```diff
@@ -14,18 +14,18 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.apisubscriptionfields
+package uk.gov.hmrc.apisubscriptionfields.config
 
-import uk.gov.hmrc.play.audit.http.config.LoadAuditingConfig
+import uk.gov.hmrc.http._
+import uk.gov.hmrc.http.hooks.HttpHook
 import uk.gov.hmrc.play.audit.http.connector.AuditConnector
-import uk.gov.hmrc.play.config.{AppName, RunMode}
-import uk.gov.hmrc.play.http.hooks.HttpHook
 import uk.gov.hmrc.play.http.ws._
+import uk.gov.hmrc.play.microservice.config.LoadAuditingConfig
 
-object WSHttp extends WSGet with WSPut with WSPost with WSDelete with WSPatch with AppName {
+object WSHttp extends WSHttp with HttpGet with HttpPost with HttpPut with HttpDelete with HttpPatch {
```

Comment:
Do we need `HttpPost` and `HttpPatch`?
We use only GET, DELETE and PUT in our routes.

Reply:
Removed as it wasn't referenced anywhere.